cmd/agent/api/grpc.go

```diff
@@ -198,6 +200,13 @@ func (s *serverSecure) GetConfigs(ctx context.Context, in *pb.GetConfigsRequest)
 		return nil, errors.New("remote configuration service not initialized")
 	}
 
+	if in.TracerInfo != nil {
+		err := s.configService.TracerInfos.AddTracer(in.TracerInfo)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
Do we want to fail the request when we can't cache the tracer info?

Maybe log a debug message and let the tracer receive data.